cns/ipampool/metrics.go

```diff
@@ -133,4 +142,9 @@ func observeIPPoolState(state ipPoolState, meta metaState) {
 	ipamPrimaryIPCount.WithLabelValues(labels...).Set(float64(len(meta.primaryIPAddresses)))
 	ipamRequestedIPConfigCount.WithLabelValues(labels...).Set(float64(state.requestedIPs))
 	ipamTotalIPCount.WithLabelValues(labels...).Set(float64(state.totalIPs))
+	var exhaustionState float64 = 0
```

Comment:
I would suggest `consts` for these states, as a reader I'm not sure what state `0` or `1` means.